tests/test_ops_unary.py

```diff
@@ -324,3 +324,12 @@ def softsign(x):
     expected = softsign(data)
     ng_results = run_node(node, [data])
     assert np.allclose(ng_results, [expected])
+
+
+def test_identity():
+    np.random.seed(133391)
+    input_data = np.random.randint(-100, 100, (2, 3, 4), dtype=np.int32)
+
+    node = onnx.helper.make_node('Identity', inputs=['x'], outputs=['y'])
```

Comment:
Could you also add a test which checks that the Identity op works in a larger model? Let's say 3 nodes, one upstream and one downstream of identity?